app/domain/errors.rb

```diff
@@ -75,6 +75,16 @@ module Authentication
       code: "CONJ00017E"
     )
 
+    RoleMissingRequiredConstraints = ::Util::TrackableErrorClass.new(
+      msg:  "Role should have at least 1 constraints: {0-constraints}",
+      code: "CONJ00067E"
+    )
```

Comment:
i think that this is a GCP authenticator restriction, so it should go under its module

Reply:
also - i would change the text to `Role should have at least one of the following constraints: {0-constraints}`